README.md

```diff
@@ -2,41 +2,39 @@
 
 This repository contains all information and tools to deploy a bridgehead. If you have any questions about deploying a bridgehead, please [contact us](mailto:verbis-support@dkfz-heidelberg.de).
 
-
-# Table of Contents
+## Table of Contents
 
 1. [About](#about)
-    - [Projects](#projects)
-        - [GBA/BBMRI-ERIC](#gbabbmri-eric)
-        - [CCP(DKTK/C4)](#ccpdktkc4)
-        - [NNGM](#nngm)
-    - [Bridgehead Components](#bridgehead-components)
-        - [Blaze Server](#blaze-serverhttpsgithubcomsamplyblaze)  
-        - [Connector](#connector) 
-1. [Requirements](#requirements)
+2. [Requirements](#requirements)
     - [Hardware](#hardware)
-    - [System](#system-requirements)
+    - [System Requrements](#system-requirements)
       - [git](#git)
       - [docker](#dockerhttpsdocsdockercomget-docker)
-      - [systemd](#systemd)
-2. [Getting Started](#getting-started)
-    - [DKTK](#dktkc4)
-    - [C4](#c4)
-    - [GBA/BBMRI-ERIC](#gbabbmri-eric)
-3. [Configuration](#configuration)
-4. [Managing your Bridgehead](#managing-your-bridgehead)
-    - [Systemd](#on-a-server)
-    - [Without Systemd](#on-developers-machine)
-4. [Pitfalls](#pitfalls)
-5. [Migration-guide](#migration-guide)
+3. [Getting Started](#getting-started)
+    - [Installation](#installation)
+4. [Configuration](#configuration)
+    - [Authentication](#basic-auth)
+        - [systemd](#systemd)
+        - [environment](#without-systemd)
+    - [Testing](#testing-your-bridgehead)
+    - [After the Installation](#after-the-installation)
+5. [Roadmap](#roadmap-🚀)
+6. [Authors](#authors)
 7. [License](#license)
-
+8. [Build With](#build-with)
+9. [Acknowledgements](#acknowledgements)
 
 ---
 
 ## About
 
-TODO: Insert comprehensive feature list of the bridgehead? Why would anyone install it?
+The Bridgehead is a collection of Software componentens for medical informatics usecases. This repository helps the user to deploy these compoents efficently. The bridgehead contains of project independen components:
```

Comment:
Maybe this would explain the purpose of this repository better:

The Bridgehead is a collection of Software componentens for medical informatics usecases. This repository aims to ease the deployment of those components to their users, by providing consistent configuration and software updates across the usecases. For this purpose, a minimal set of default components is integrated within this repository:
- A **forward proxy**, as a central point for scanning and logging outgoing network traffic of the components deployed with the bridgehead
- A **reverse proxy**, that distributes incoming network traffic to the components of the bridgehead.
- A **landingpage**, that offers a quick graphical overview of all components running in the bridgehead
  
Because the different use cases of the bridgehead all require different components, we provide detailed project specific documentation on these inside [Projects](docs/projects.md).